pyspark-notebook/Dockerfile

```diff
@@ -46,19 +46,21 @@ USER jovyan
 
 # Install Python 3 packages
 RUN conda install --quiet --yes \
-    'ipywidgets=4.1*' \
+    'ipywidgets=5.1*' \
     'pandas=0.17*' \
     'numexpr=2.5*' \
     'matplotlib=1.5*' \
     'scipy=0.17*' \
     'seaborn=0.7*' \
     'scikit-learn=0.17*' \
     && conda clean -tipsy
-
+# Activate ipywidgets extension in the environment that runs the notebook server
+RUN jupyter nbextension enable --py widgetsnbextension --sys-prefix
+    
 # Install Python 2 packages and kernel spec
 RUN conda create --quiet --yes -p $CONDA_DIR/envs/python2 python=2.7 \
-    'ipython=4.1*' \
-    'ipywidgets=4.1*' \
+    'ipython=4.2*' \
+    'ipywidgets=5.1*' \
```

Comment:
5.1.2 is already out, hot on the heels of 5.0: https://github.com/ipython/ipywidgets/releases

I figured we'd go to the latest.